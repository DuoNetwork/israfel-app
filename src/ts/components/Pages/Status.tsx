import * as React from 'react';
import { IStatus } from 'ts/common/types';
import StatusCard from '../Cards/StatusCard';

interface IProps {
	status: IStatus[];
}

export default class Status extends React.Component<IProps> {
	public componentDidMount() {
		document.title = 'DUO | Status';
	}

	public render() {
		const { status } = this.props;
		return (
			<StatusCard status={status} />
		);
	}
}
