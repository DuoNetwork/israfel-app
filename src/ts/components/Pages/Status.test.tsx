/**
 * @jest-environment jsdom
 */

import { shallow } from 'enzyme';
import * as React from 'react';
import Status from './Status';

describe('Status Test', () => {
	describe('User Login', () => {
		it('Test Snapshot', () => {
			const status = [
				{
					hostname: 'test',
					updatedAt: 123,
					pair: 'test',
					tool: 'test',
					count: 123
				}
			];
			const wrapper = shallow(<Status status={status} />);
			expect(wrapper).toMatchSnapshot();
		});
	});
});
