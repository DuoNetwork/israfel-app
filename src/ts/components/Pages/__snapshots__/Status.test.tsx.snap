// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Status Test User Login Test Snapshot 1`] = `
<StatusCard
  status={
    Array [
      Object {
        "count": 123,
        "hostname": "test",
        "pair": "test",
        "tool": "test",
        "updatedAt": 123,
      },
    ]
  }
/>
`;
